packages/cmr-client/getUrl.js

```diff
@@ -11,7 +11,7 @@ function hostId(env) {
   return get(
     { OPS: '', SIT: 'sit', UAT: 'uat' },
     env,
-    'uat'
+    'sit'
```

Comment:
Jenny recommended we use sit here.

Reply:
If we change cmr-client to use `sit` as default, should we change cmrjs to use `sit` as default as well?